Formula/conduit.rb

```diff
@@ -0,0 +1,30 @@
+class Conduit < Formula
+  desc "Streams data between data stores. Kafka Connect replacement. No JVM required"
+  homepage "https://conduit.io/"
+  url "https://github.com/ConduitIO/conduit",
+   using:    :git,
+   tag:      "v0.5.0",
+   revision: "eacafff5bc575f14396a95c8fd402b8316c1dfb2"
```

Comment:
```suggestion
  url "https://github.com/ConduitIO/conduit.git",
      tag:      "v0.5.0",
      revision: "eacafff5bc575f14396a95c8fd402b8316c1dfb2"
```

Reply:
@carlocab I changed it to a clone because the command `conduit -version` depends on the git tags to get the version, so installing it using a tarball made the version test fail.
is it preferred to have a tarball for a specific reason?

Reply:
You can use a tarball if you call
```ruby
# Use formula version instead of relying on Git tags
inreplace "Makefile" { |s| s.change_make_var! "VERSION", version }
```
before calling `make`.

Reply:
@carlocab done! thank you for the suggestion.
do you have any idea why "open3" is not loaded in some tests?